docs/source/fractal-procedures.rst

```diff
@@ -0,0 +1,2 @@
+Procedures
```

Comment:
Note that the file names become the html names (e.g., `fractal-procedures.html`) and ppl link to those. Fine by me, just a heads-up if you want fractal everywhere.

Reply:
I go back and forth on this topic, not really sure which one I like better. Portal and Fractal are divided on these lines for experimentation.